pkg/crypto/webkms/wrapkey_util.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webkms
+
+import (
+	"encoding/base64"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto"
+)
+
+// WrapKeyReq serializable WrapKey request.
+type WrapKeyReq struct {
+	CEK       string       `json:"cek,omitempty"`
+	APU       string       `json:"apu,omitempty"`
+	APV       string       `json:"apv,omitempty"`
+	RecPubKey PublicKeyReq `json:"recpubkey,omitempty"`
+	SenderKID string       `json:"senderkid,omitempty"`
+}
+
+// WrapKeyResp serializable WrapKey response.
+type WrapKeyResp struct {
+	WrappedKey RecipientWrappedKeyReq `json:"WrappedKey,omitempty"`
```

Comment:
WrappedKey => wrappedKey

Reply:
good catch in the json definition, it needs to start with a small w.. I'll update

And as discussed all functions and structs in wrap key_util.go will become unexported as they're going to be used internally only.

Reply:
done